.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,31 @@
+name: Python application
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
```

Comment:
I want to support 3.6 (because it's included in most distros, at least more than 3.7). So it's better to test on 3.6

Reply:
GitHub actions has a matrix feature, which allows testing arbitrary interpreter versions with minimal configuration overhead. Since we have now verified Python 3.6 and 3.7 to work by accident. so I activated them both in the actions file.